src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
```

Comment:
Ensure that the `Injector` class is correctly configured to handle the package `mate.academy.dao`. This is crucial for dependency injection to work properly.